datadog/resource_datadog_monitor.go

```diff
@@ -559,41 +559,8 @@ func resourceDatadogMonitorCreate(d *schema.ResourceData, meta interface{}) erro
 	return resourceDatadogMonitorRead(d, meta)
 }
 
-func resourceDatadogMonitorRead(d *schema.ResourceData, meta interface{}) error {
+func updateMonitorState(d *schema.ResourceData, meta interface{}, m *datadogV1.Monitor) error {
 	providerConf := meta.(*ProviderConfiguration)
```

Comment:
is providerConf still used in this method?

Reply:
yes, to get the time near the end